resources/services/glue/registries.go

```diff
@@ -0,0 +1,244 @@
+						Type:        schema.TypeBigInt,
+					},
+					{
+						Name:        "registry_arn",
+						Description: "The Amazon Resource Name (ARN) of the registry.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "registry_name",
+						Description: "The name of the registry.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "arn",
+						Description: "The Amazon Resource Name (ARN) of the schema.",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("SchemaArn"),
+					},
+					{
+						Name:        "schema_checkpoint",
+						Description: "The version number of the checkpoint (the last time the compatibility mode was changed).",
+						Type:        schema.TypeBigInt,
+					},
+					{
+						Name:        "schema_name",
+						Description: "The name of the schema.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "schema_status",
+						Description: "The status of the schema.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "updated_time",
+						Description: "The date and time the schema was updated.",
+						Type:        schema.TypeString,
+					},
+				},
+			},
+		},
+	}
+}
+
+// ====================================================================================================================
+//                                               Table Resolver Functions
+// ====================================================================================================================
+
+func fetchGlueRegistries(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	cl := meta.(*client.Client)
+	svc := cl.Services().Glue
+	input := glue.ListRegistriesInput{MaxResults: aws.Int32(100)}
+	for {
+		result, err := svc.ListRegistries(ctx, &input)
+		if err != nil {
+			return diag.WrapError(err)
+		}
+		res <- result.Registries
+		if aws.ToString(result.NextToken) == "" {
+			break
+		}
+		input.NextToken = result.NextToken
+	}
+	return nil
+}
+func resolveGlueRegistryTags(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
+	cl := meta.(*client.Client)
+	svc := cl.Services().Glue
+	r := resource.Item.(types.RegistryListItem)
+	result, err := svc.GetTags(ctx, &glue.GetTagsInput{
+		ResourceArn: r.RegistryArn,
+	})
+	if err != nil {
+		if cl.IsNotFoundError(err) {
+			return nil
+		}
+		return diag.WrapError(err)
+	}
+	return diag.WrapError(resource.Set(c.Name, result.Tags))
+}
+func fetchGlueRegistrySchemas(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	r := parent.Item.(types.RegistryListItem)
+	cl := meta.(*client.Client)
+	svc := cl.Services().Glue
+	input := glue.ListSchemasInput{
+		RegistryId: &types.RegistryId{RegistryArn: r.RegistryArn},
+		MaxResults: aws.Int32(100),
+	}
+	for {
+		result, err := svc.ListSchemas(ctx, &input)
```

Comment:
There is an automated way of `list - get` process. it should be used here. Look at `client.ListAndDetailResolver`